catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/ReplaceExistingProjectDialogFragment.java

```diff
@@ -86,33 +84,30 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
 
 			@Override
 			protected boolean isNameUnique(String name) {
-				return projectExistsInDirectory(name) || GlobalProjectDownloadQueue.INSTANCE.getQueue().alreadyInQueue(name);
+				return !projectExistsInDirectory(name) || GlobalProjectDownloadQueue.INSTANCE.getQueue().alreadyInQueue(name);
```

Comment:
I think there is still some kind of inversed logic here. Either you negate the whole term or you have to use 
`return !projectExistsInDirectory(name) && !GlobalProjectDownloadQueue.INSTANCE.getQueue().alreadyInQueue(name);` which is basically equivalent to that.